components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/keystore/KeyStoreAdmin.java

```diff
@@ -819,6 +905,96 @@ public KeyStore getTrustStore() throws SecurityConfigException {
         return trustStore;
     }
 
+    /**
+     * Fill PaginatedKeyStoreData with keystore details.
+     *
+     * @param keyStoreName Name of the keystore.
+     * @param keyStoreType Type of the keystore.
+     * @param certs        Paginated certificates.
+     * @param keyCerts     Paginated key certificates.
+     * @return Paginated KeyStore Data.
+     */
+    private PaginatedKeyStoreData fillPaginatedKeyStoreData(String keyStoreName, String keyStoreType,
+                                                            PaginatedCertData certs, PaginatedCertData keyCerts) {
+
+        // Create a KeyStoreData bean, set the name, type and fill in the cert information.
+        PaginatedKeyStoreData keyStoreData = new PaginatedKeyStoreData();
+        keyStoreData.setKeyStoreName(keyStoreName);
+        keyStoreData.setKeyStoreType(keyStoreType);
+        keyStoreData.setPaginatedCertData(certs);
+        keyStoreData.setPaginatedKeyData(keyCerts);
+        return keyStoreData;
+    }
+
+    /**
+     * Get list of certificates from aliases.
```

Comment:
Change method description to "Get certificates related to alias from the keystore".
Change method name "getCertificates"

Apply same to the getKeyCertificates method.

Reply:
Resolved with PR https://github.com/wso2/carbon-identity-framework/pull/4125